README.md

```diff
@@ -15,7 +15,13 @@ file at the root of your package, like so:
   "license": "(MIT OR BSD-2-Clause OR BSD-3-Clause OR Apache-2.0)",
   "whitelist": {
     "optimist": "<=0.6.1"
-  }
+  },
+  "corrections": false,
+  "ignore": [
+    {"scope": "kemitchell"},
+    {"prefix": "commonform-"},
+    {"author": "Kyle E. Mitchell"}
```

Comment:
what about an author's email address, or when "author" is the object form?

Reply:
Keep reading. https://github.com/jslicense/licensee.js/pull/39/files#diff-168726dbe96b3ce427e7fedce31bb0bcR282

Reply:
Sweet!